bddl/activity_definitions/make-chicken-and-waffles/problem0.bddl

```diff
@@ -54,7 +54,7 @@
         (filled hot_sauce__bottle.n.01_1 hot_sauce.n.01_1)
         (inside hot_sauce__bottle.n.01_1 electric_refrigerator.n.01_1)
         (ontop plate.n.04_1 countertop.n.01_1)
-        (filled tupperware.n.01_1 batter.n.02_1)
+        (filled tupperware.n.01_1 fritter_batter.n.01_1)
```

Comment:
fritter batter is an existing WN synset, which is used for frying stuff I think. In this task, I believe it's the coating for the fried chicken (instead of for the waffles), so I thought fritter batter is more appropriate than pancake batter.